setup.cfg

```
@@ -9,8 +9,10 @@ description-content-type = text/x-rst; charset=UTF-8
 zip_safe = false
 install_requires =
     canonicaljson
-    connexion[swagger-ui]
+    fastapi
+    uvicorn
     ga4gh.vrs[extras]==0.7.6
+    ga4gh.vrsatile.pydantic ~= 0.0.11
```

Without this I got this error:
```
ERROR:    Traceback (most recent call last):
  File "biocommons/anyvar/venv/lib/python3.10/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "biocommons/anyvar/venv/lib/python3.10/site-packages/starlette/routing.py", line 566, in __aenter__
    await self._router.startup()
  File "biocommons/anyvar/venv/lib/python3.10/site-packages/starlette/routing.py", line 654, in startup
    await handler()
  File "biocommons/anyvar/src/anyvar/restapi/main.py", line 27, in startup
    storage = create_storage()
  File "biocommons/anyvar/src/anyvar/anyvar.py", line 70, in create_storage
    from anyvar.storage.postgres import PostgresObjectStore
  File "biocommons/anyvar/src/anyvar/storage/postgres.py", line 4, in <module>
    import psycopg
ModuleNotFoundError: No module named 'psycopg'
```

```suggestion
    ga4gh.vrsatile.pydantic ~= 0.0.11
    psycopg
```